src/vma/iomux/epoll_wait_call.cpp

```diff
@@ -79,7 +79,7 @@ int epoll_wait_call::get_current_events()
 		return m_n_all_ready_fds;
 	}
 
-	vector<socket_fd_api *> socket_fd_vec;
+	vma_list_t<socket_fd_api, socket_fd_api::socket_fd_vec_node_offset> socket_fd_vec;
```

Comment:
For all cases of ..._socket_fd_ver_... names - change to ..._socket_fs_list_... names.
For example: change variable name socket_fd_vec to socket_fd_list